deployments/warp_routes/PZETH/ethereum-zircuit-config.yaml

```
@@ -0,0 +1,20 @@
+# yaml-language-server: $schema=../schema.json
+tokens:
+  - addressOrDenom: "0x0220b1EA1b56ECE8e2b62c8965659f0A621e9ebd"
+    chainName: ethereum
+    collateralAddressOrDenom: "0xbC5511354C4A9a50DE928F56DB01DD327c4e56d5"
+    connections:
+      - token: ethereum|zircuit|0x8303Ce0E207BB44a3D3B2313AC219d0fC73b3764
+    decimals: 18
+    name: "Renzo Restaked LST "
```

trailing space on the names?
```suggestion
    name: "Renzo Restaked LST "
```

---

Think this is from their token contract name. i'll leave for now unless we need to change it